lib/quadquizaminos/bottom.ex

```diff
@@ -14,6 +14,13 @@ defmodule Quadquizaminos.Bottom do
   def collides?(bottom, points) when is_list(points) do
     Enum.any?(points, &collides?(bottom, &1))
   end
+   
+  def vulnerable?(bottom, row) do
+    bottom
```

Comment:
```suggestion
    bottom
    |> Enum.map(fn {_key, value} -> value end)
    |> Enum.any?(fn {_x, y, color} -> y == row and color == :vuln_grey_yellow end)
```
I have tried this out, you can check for vulnerability in the bottom in a specific row using Enum.any?. You can eliminate Enum.filter